service/static/lr-styleguide/css/styleguide.css

```diff
@@ -1450,3 +1450,41 @@ input[type=number] {
 .service-notice p {
     margin-bottom: 0
 }
+
+.language {
+  font-family: "nta", Arial, sans-serif;
+  font-size: 14px;
+  line-height: 1.14286;
+  font-weight: 400;
+  text-transform: none;
+  padding: 0.75em 0 0 0;
+}
+
+@media (min-width: 641px) {
+  .language {
+    font-size: 14px;
+    line-height: 1.25;
+  }
+}
+
+@media (min-width: 641px) {
```

Comment:
Minor point, but this could be merged with the block above since they both target `.language` within the same media query range.